addons/mail/static/src/messaging/component/composer/composer.js

```diff
@@ -263,17 +263,12 @@ class Composer extends Component {
      * Called when clicking on "send" button.
      *
      * @private
-     * @param {MouseEvent} ev
      */
-    _onClickSend(ev) {
-        if (
-            !this.composer.textInputContent &&
-            this.composer.attachments.length === 0
-        ) {
+    _onClickSend() {
+        if (!this.composer.canPostMessage) {
```

Comment:
same question here, canPostMessage depends on textInputContent, which must be correctly saved in entity prior to this call in order to be correctly computed